react/components/CategoriesHighlights/README.md

```diff
@@ -1,31 +1,64 @@
-# CategoriesHighlights description
-CategoriesHighlights is a canonical component that any VTEX app can import.
+# Categories Highlights
+
+## Description
+
+Categories Highlights shows two or four categories in a lighlighted position in the store. This Component can be imported and used by any VTEX App.
```

Comment:
`lighlighted `?

Reply:
What do you think?

```suggestion
Categories Highlights shows two or four categories, in the form of an image, with a highlighted position in the store. This Component can be imported and used by any VTEX App.
```

Reply:
I wrote it this way: `Categories Highlights is a banner which shows two or four categories in a highlighted position in the store. This Component can be imported and used by any VTEX App.`

by saying it is a banner we imply that it has an image in it.